benchexec/tablegenerator/__init__.py

```diff
@@ -697,6 +695,13 @@ def read_logfile_lines(log_file):
 
                     value = get_value_from_logfile(logfileLines, column.pattern)
 
+            if column.title.lower() == 'score':
+                if value is None:
+                    value = str(score)
+                else:
+                    # If a score column exists in the xml, we use that column's value for the score, instead of
+                    # the computed score.
+                    score = float(value)
```

Comment:
> Why is score a float here?

Scores are floats, not integers. Mind the normalization and coverage scores.

Reply:
> What happens if there is a score column, but it contains text? Won't the current code crash table-generator?

Add exception handling to the float conversion?

Reply:
I missed the exception handling, but also chose "float" because I had the notion in mind that scores must be numbers.
But if a user decides to store some other values in a column called score, should this be forbidden?

When a user actually uses numbers, this decision won't make any difference: Even if we choose the string-Type here, if the score is a number, it should be correctly recognized by table-generator during output formatting.

Reply:
I've added exception handling and a small integration test.